content/html/concepts/tags/terms/embed/embed.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: '<embed>'
+Description: 'The <embed> tag allows you to "Embed" a video, image, page and more.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Images'
+  - 'Video'
+  - 'HTML'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option
+
+<!--  -->
+
+## Syntax
+
+```html
+<embed type="image/jpg" src="my_image.jpg" width="300" height="200" />
+```
```

Comment:
Can even add a **Note:** here that this tag is deprecated so we are covering it for legacy purposes.

Reply:
It would also be great to add that this is a self-closing tag (also referred to as void or empty tags).